plugin.video.invidious/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.invidious"
+  name="Invidious"
+  version="0.2.2+nexus.0"
+  provider-name="petterreinholdtsen">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+        <import addon="script.module.inputstreamhelper" version="0.5.2"/>
+    </requires>
+
+    <extension point="xbmc.python.pluginsource" library="resources/lib/invidious_addon.py">
+        <provides>video</provides>
+    </extension>
+
+    <extension point="xbmc.python.module" library="resources/lib/"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Invidious client</summary>
```

Comment:
Please change all `lang="en"` to the default `lang="en_GB"` THX

Reply:
Thank you for the feedback.  Changed upstream.